doc/user/bgp.rst

```diff
@@ -1816,6 +1816,12 @@ Configuring Peers
 
    Send the BGP Next Hop Dependent Characteristics Attribute (NHC) to the peer.
 
+   If the route is learned and NHC attribute is present with Next-Next Hop Nodes
+   (NNHN) TLV set, then the number of NNHN can be used for weighted ECMP.
+
+   In order to use weighted ECMP, ``bgp bestpath bandwidth ignore`` must be
```

Comment:
Can we say something like `In order to use weighted ECMP with NNHN, ``bgp....`. The first time I read this the implication to me was that it was the only way to make it work.